### PR TITLE
fhir documention update for financial transaction

### PR DESCRIPTION
**Pull request by Anurag-Balannagari**
Description
----

- Updating examples to use real financial-transaction-location ids from C1941. location id 98920358
image
image
image
image

- Updating examples to use real encounter ids for allocation.target from C1941. Encounter/97953536
image
image
image
image
image
image

- Updating payment examples to use real financial-transaction-alias ids from C1941. alias 0111 (Note that the adjustment example should stay the same.)
image
image
image
image

- Removing the implementation note of "A FinancialTransaction can only be targeted at a clinical Encounter that has related financial activity/encounters within the Cerner Revenue Cycle Patient Accounting system.”
- Removing the word "production" from the implementation note so it reads "When integrating your application with a client’s environment, the client will have to provide the appropriate financial-transaction-alias values to send with the financial transaction."
image

- Removing "EFT date if method is EFT or Lockbox date if method is lockbox." from the financial-transaction-date extension description.
image

- Updating the body field Notes for the financial-transaction-alias extension to: A value that correlates to a client defined name which represents the combination of the type, subtype, and reason describing the FinancialTransaction. The alias determines the credit or debit impact on the accounts receivable. A valid alias from the tenant is required to create a FinancialTransaction resource.
image

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: content/millennium/r4/financial/financial-transaction.md [hunk 1]
```diff
@@ -35,11 +35,11 @@ All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/St
  ID                                      | Value\[x] Type                    | Description
 -----------------------------------------|-----------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------
  `financial-transaction-account-number`  | [`string`]                        | A value associated to the specific payment method usually represented as the last four digits of a credit card, the check number, the EFT number or Lockbox number.
- `financial-transaction-alias`           | [`string`]                        | Client defined value to represent the combination of the type, subtype, and reason describing the financial transaction.
+ `financial-transaction-alias`           | [`string`]                        | Client defined value to represent the combination of the type, subtype, and reason describing the financial transaction. The alias needs to be a valid alias in the client domain for the transaction type and that the alias determines the credit or debit impact on the accounts receivable.
```
Comment (sujanb12) [inline content/millennium/r4/financial/financial-transaction.md:38]: Why is this updated?

Comment (Anurag-Balannagari) [reply content/millennium/r4/financial/financial-transaction.md]: removing this. Initially change made here but this was not expected change.
https://github.com/cerner/fhir.cerner.com/pull/455/commits/0017aa837e886e8b5c88cc8bf114295d75a54f2e#diff-a2a1182a3c7ddb3de2fa0458e90ddf0cR38

Comment (sujanb12) [reply content/millennium/r4/financial/financial-transaction.md]: +1

**Review by sujanb12 — APPROVED** [review]

File: content/millennium/r4/financial/financial-transaction.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/St
  `financial-transaction-allocation`      | None (contains nested extensions) | Defines how the payment or adjustment is to be allocated across other resources.
  `financial-transaction-amount`          | [`Money`]                         | The total amount of the financial transaction.
  `financial-transaction-card-brand`      | [`string`]                        | Identifies the brand of credit card when credit card is used as a payment method.
- `financial-transaction-date`            | [`date`]                          | Represents the expiration date if method is card, check date if method is check, EFT date if method is EFT or Lockbox date if method is lockbox.
+ `financial-transaction-date`            | [`date`]                          | Represents the expiration date if method is card, check date if method is check.
```
Comment (sydneanschutz) [inline content/millennium/r4/financial/financial-transaction.md:42]: Can this read 'Represents the expiration date if method is card or check date if method is check.'

File: lib/resources/example_json/r4_examples_financial_transaction.rb [hunk 3]
```diff
@@ -131,11 +131,11 @@ module Resources
         },
         {
           'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-alias',
-          'valueString': '1300'
+          'valueString': '0111'
```
Comment (sydneanschutz) [inline lib/resources/example_json/r4_examples_financial_transaction.rb:134]: should line 54 be this new alias too?

Comment (Anurag-Balannagari) [reply lib/resources/example_json/r4_examples_financial_transaction.rb:134]: Reason for not changing this example, It was mentioned that adjustment example should not be changed and remain same in the Jira.

Comment (sydneanschutz) [reply lib/resources/example_json/r4_examples_financial_transaction.rb:134]: Gotcha +1

File: content/millennium/r4/financial/financial-transaction.md (hunk 2 above)
Comment (Anurag-Balannagari) [reply content/millennium/r4/financial/financial-transaction.md]: updated here.
https://github.com/cerner/fhir.cerner.com/pull/455/commits/a19a9a85fb67aaadc8eb7d2893b02d8363b09316#diff-a2a1182a3c7ddb3de2fa0458e90ddf0cR42

**Review by sydneanschutz — APPROVED** [review]

**Review by MadhurRaj — APPROVED** [review]

**Comment by MaxPhilips** [comment]
verified on live

Screen Shot 2020-09-24 at 5 57 20 PM

**Pull request merged**